src/site/_includes/layout.njk

```diff
@@ -171,6 +170,7 @@ eleventyComputed:
         {% if show_banner %}
           {% include 'partials/banner.njk' %}
         {% endif %}
+        {% include 'partials/translation-disclaimer.njk' %}
```

Comment:
I was only able to trigger this banner by going to localhost:8080/pl/

Here's the code for the partial:
```
{% set gitlocalizeUrl =  page.inputPath | gitlocalizeLink(lang) %}
{% if translation %}
{% Banner 'neutral' %}
    {% if translation === 'none' %}
    This page is not translated yet.
    {% elif translation === 'machine' %}
    This page is automatically translated using the Google Translation
    API.
    {% elif translation === 'manual' %}
    This article was translated from
    [the original post](/en{{ page.data.canonicalUrl }}).
    {% endif %}
    {% if gitlocalizeUrl %}
    Consider improving it by [contributing your translation]({{ gitlocalizeUrl }}).
    {% endif %}
{% endBanner %}
{% endif %}
```

- Since we redirect articles with missing translations would they ever show the `'none'` condition?
- Why does the homepage show the `'none'` condition but articles don't? Should the homepage do a similar redirect?

Reply:
This is because we don't have a pl translation folder fully checked in to the repo. Please use stage (https://web-dev-staging.uc.r.appspot.com/pl/measure/) instead of local dev env and you should see banner everywhere in /pl/

Reply:
ah I see